tests/adopt-volume_test.go

```diff
@@ -0,0 +1,218 @@
+package tests_test
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	cdiv1 "kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+	controller "kubevirt.io/containerized-data-importer/pkg/controller/common"
+	featuregates "kubevirt.io/containerized-data-importer/pkg/feature-gates"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+var _ = Describe("PVC adoption tests", Serial, func() {
```

Comment:
these have to be serial just because of the feature gate, right?

Reply:
according to that block's before/after each it will flip the feature gate too, and thus potentially mess up other tests. what am I missing

Reply:
Yeah good catch, forgot about that, doesn't actually flip the featuregate now but will if/when it is the default